conf/data.sql.j2

```diff
@@ -1,20 +1,14 @@
-INSERT INTO `lime_plugins` (`id`, `name`, `active`) VALUES 
+INSERT INTO `lime_plugins` (`id`, `name`, `active`) VALUES
 (2,'AuditLog',0),
 (3,'oldUrlCompat',0),
 (4,'ExportR',0),
 (5,'Authwebserver',1),
 (6,'extendedStartPage',0),
 (7,'ExportSTATAxml',0),
 (8,'QuickMenu',0),
-(9,'AuthLDAP',0);
+(9,'AuthLDAP',1);
```

Comment:
Yes, but there are some issue with Auth webserver if LS are not a subdirectory of ynh primary server. Last time i test it ....